### PR TITLE
add databricks soln accelerator to case studies

### PR DESCRIPTION
**Pull request by fverac**
Small PR to add Databricks solution accelerator to PyWhy website. Currently added under Case Studies section. Attaching screenshots of changes:

image
image

### CODE REVIEW

**Review by amit-sharma — DISMISSED** [review]
Looks good, thanks

File: _case_studies/09_databricks_solution_accelerator.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Optimizing Promotional Offers using Causal Machine Learning
```
Comment (emrekiciman) [inline _case_studies/09_databricks_solution_accelerator.md]: Let's add databricks to the title.  Compared to other case studies on pywhy, the fact that this is implemented in databricks is a differentiator that should be front and center

**Review by emrekiciman — CHANGES_REQUESTED** [review]

File: _case_studies/09_databricks_solution_accelerator.md (hunk 1 above)
Comment (fverac) [reply _case_studies/09_databricks_solution_accelerator.md]: Good point. Which of the following do you prefer:

"Optimizing Promotional Offers using Causal Machine Learning in Databricks"
"Using PyWhy in Databricks"
"Databricks Causal Machine Learning Solution Accelerator"
"PyWhy in the Databricks ecosystem"

Or happy to take suggestions on title.

Comment (emrekiciman) [reply _case_studies/09_databricks_solution_accelerator.md]: I like the second one, "Using PyWhy in Databricks", at least until there are multiple examples :)

**Review by emrekiciman — APPROVED** [review]

File: _case_studies/09_databricks_solution_accelerator.md (hunk 1 above)
Comment (fverac) [reply _case_studies/09_databricks_solution_accelerator.md]: Updated, thanks!
image

**Comment by fverac** [comment]
If we're all set, someone with write access to the repo can merge.

**Pull request merged**